spec/spec_helper.rb

```diff
@@ -67,15 +68,12 @@
   # the seed, which is printed after each run.
   #     --seed 1234
 
-  puts "Randomized with seed #{config.seed}."
-
```

Comment:
Is this printed out by default now? I find it useful.

Reply:
Yes @enricostano, I saw it duplicated now.

Reply:
Weirdly enough I don't see it duplicated in `develop` :thinking:

Reply:
Maybe you have a global `.rspec` with any kind of setting preventing this to be displayed?

Reply:
Not that I am aware of :joy:
```
timeoverflow@timeoverflow:/var/www/timeoverflow$ ls ~/.rspec
ls: cannot access '/home/timeoverflow/.rspec': No such file or directory
timeoverflow@timeoverflow:/var/www/timeoverflow$ ls ./.rspec
ls: cannot access './.rspec': No such file or directory
```

Reply:
Ok @enricostano, it's my global `.rspec` 😂

```
[2.3.0] markets: ~/workspace/timeoverflow (develop)$ more ~/.rspec 
--color
--order random
--format documentation
```

This is printed by default by `RSpec` if you have `config.order = "random"`, but this was removed here: https://github.com/coopdevs/timeoverflow/pull/333/commits/94c50fbec07ae4fcecfd7d31914456d42d5b1dc1. I'm going to put it back :sweat_smile: